code/modules/mob/living/simple_animal/abnormality/_tools/he/treesap.dm

```diff
@@ -24,14 +24,14 @@
 	if(!do_after(user, 6))
```

Comment:
```suggestion
	if(!do_after(user, 6, src))
```
This will prevent extremely comical event that is multiple sips at once. This event also allows the person to potentially accidentally roll saps multiple times potentially resulting in the fabled "Boom Sap"